src/moduler/aktivitet/ikke-delt-markering/IkkeDeltMarkering.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import EtikettBase from '../../../felles-komponenter/etikett-base/etikett-base';
+import styles from './ikke-delt-markering.module.less';
+import visibleIfHOC from "../../../hocs/visible-if";
+
+const IkkeDeltMarkering = () => {
+    return (
+        <EtikettBase className={styles.etikett}>
+            Samtalereferatet er ikke delt
+        </EtikettBase>
+    );
+}
```

Comment:
Nå kan denne returneres direkte siden det er en arrow function 😊 

```suggestion
const IkkeDeltMarkering = () => (
        <EtikettBase className={styles.etikett}>
            Samtalereferatet er ikke delt
        </EtikettBase>
    );
```